packages/fx-core/resource/package.nls.json

```diff
@@ -301,6 +301,10 @@
   "core.createProjectQuestion.projectType.customCopilot.label": "Custom Copilot",
   "core.createProjectQuestion.projectType.customCopilot.title": "App Features Using Teams AI Library",
   "core.createProjectQuestion.projectType.customCopilot.placeholder": "Select an option",
+  "core.createProjectQuestion.projectType.copilotHelp.label": "Don't know how to start? Use Github Copilot Chat",
+  "core.createProjectQuestion.projectType.copilotHelp.detail": "Chat with Github Copilot to get step by step instructions for Teams app development",
```

Comment:
Proposed: Chat with GitHub Copilot and get step-by-step instructions to develop your Teams app